website/src/app/pricing/_page.tsx

```diff
@@ -52,7 +52,7 @@ export default function _Page() {
         </div>
         <div className="mx-auto max-w-screen-2xl md:grid md:grid-cols-3 pt-14 md:gap-4 px-4">
           <div className="p-8 bg-neutral-50 border-2 border-neutral-200">
-            <h3 className="mb-4 text-2xl tracking-tight font-semibold text-primary-450">
+            <h3 className="mb-4 text-2xl tracking-tight font-medium text-neutral-900">
```

Comment:
Was the `font-medium` intentional or did you mean to put `font-bold` like the Team and Enterprise plans?

Reply:
Intentional, can revert it

Reply:
Gotcha.  Just wasn't sure since the other two headers were bold.  Hope this wasn't the reason for closing this PR.  I'm happy to approve it, just wanted to check since it was just a one word change.